apps/condo/domains/organization/integrations/sbbol/sync/syncUser.js

```diff
@@ -74,6 +74,7 @@ const syncUser = async ({ context: { context, keystone }, userInfo, identityId }
     const identityWhereStatement = {
         identityId,
         identityType: SBBOL_IDP_TYPE,
+        deletedAt: null,
```

Comment:
You don't need to use deleted at?

Reply:
<img width="1095" alt="Screenshot 2023-02-01 at 12 50 12" src="https://user-images.githubusercontent.com/64303474/215982916-f0e92253-e20a-43dd-9aba-37e390cc193a.png">

Reply:
without it I get a deleted instance

Reply:
Looks like a bug! You should not use it